subprojects/emcee/action/src/main/kotlin/com/avito/emcee/EmceeTestActionFactory.kt

```diff
@@ -8,8 +8,8 @@ import com.avito.emcee.queue.QueueApi
 import com.avito.emcee.queue.QueueApi.Companion.create
 import retrofit2.Retrofit
 
-public class EmceeTestActionFactory {
-    private val queueApi: QueueApi = Retrofit.Builder().create()
+public class EmceeTestActionFactory(emceeQueueBaseUrl: String) {
```

Comment:
We could use URL here to validate input

Reply:
Well, it will be validated on lower level, during service construction
Whis layer doesn't have direct access to okhttp classes, i decided not to broaden it's visibility just for that

Reply:
Ok, I meant `java.net.URL` rather than okhttp classes.